Frequent failures of PDF manual build on Pyrolyse.

I frequently see the following failure in the build of the PDF manual:

```
1210 Latexmk: Run number 1 of rule 'pdflatex'
1211 This is XeTeX, Version 3.14159265-2.6-0.99998 (TeX Live 2017/Debian) (preloaded format=xelatex)
1212  restricted \write18 enabled.
1213 entering extended mode
1214 Latexmk: Summary of warnings:
1215   Latex failed to resolve 930 reference(s)
1216   Latex failed to resolve 59 citation(s)
1217 Latexmk: Run number 1 of rule 'makeindex CoqRefMan.idx'
1218 Latexmk: calling xindy( -L english -C utf8 -M sphinx.xdy -I xelatex -q -o CoqRefMan.ind CoqRefMan.idx )
1219 Cannot map memory to address 0x8023d0894000 .
1220 [/build/clisp-u7VYrq/clisp-2.49.20170913/src/spvw_mmap.d:498] errno = 12 (Collected error summary (may duplicate other messages):
1221   makeindex CoqRefMan.idx: Command for 'makeindex CoqRefMan.idx' gave return code 1024
1222 Latexmk: Use the -f option to force complete processing,
1223  unless error was exceeding maximum runs of latex/pdflatex.
1224 Makefile:35: recipe for target 'CoqRefMan.pdf' failed
1225 make[1]: *** [CoqRefMan.pdf] Error 12
1226 make[1]: Leaving directory '/builds/coq/coq/_build/default/doc/refman-pdf'
1227 make: *** [refman-pdf] Error 1
1228 Makefile.dune:103: recipe for target 'refman-pdf' failed
```

(from https://gitlab.com/coq/coq/-/jobs/543104449)

I have always observed that the runner was pyrolyse when such error happened.

Note that the converse is not true, here is a job on pyrolyse that was successful: https://gitlab.com/coq/coq/-/jobs/541601113.

On the last 3000 jobs that ran, only 4 of them were PDF manual builds on pyrolyse. 3 out of 4 failed with this error.

--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -18,7 +18,7 @@ stages:
 variables:
   # Format: $IMAGE-V$DATE [Cache is not used as of today but kept here
   # for reference]
-  CACHEKEY: "bionic_coq-V2020-05-19-V33"
+  CACHEKEY: "bionic_coq-V2020-05-24-V1"
   IMAGE: "$CI_REGISTRY_IMAGE:$CACHEKEY"
   # By default, jobs run in the base switch; override to select another switch
   OPAM_SWITCH: "base"

--- a/dev/ci/docker/bionic_coq/Dockerfile
+++ b/dev/ci/docker/bionic_coq/Dockerfile
@@ -1,4 +1,4 @@
-# CACHEKEY: "bionic_coq-V2020-05-19-V33"
+# CACHEKEY: "bionic_coq-V2020-05-24-V1"
 # ^^ Update when modifying this file.
 
 FROM ubuntu:bionic
@@ -13,8 +13,7 @@ RUN apt-get update -qq && apt-get install --no-install-recommends -y -qq \
         libgtksourceview-3.0-dev \
         # Dependencies of stdlib and sphinx doc
         texlive-latex-extra texlive-fonts-recommended texlive-xetex latexmk \
-        xindy python3-pip python3-setuptools python3-pexpect python3-bs4 \
-        fonts-freefont-otf \
+        python3-pip python3-setuptools python3-pexpect python3-bs4 fonts-freefont-otf \
         # Dependencies of source-doc and coq-makefile
         texlive-science tipa
 

--- a/doc/README.md
+++ b/doc/README.md
@@ -58,7 +58,6 @@ additional tools are required:
   - makeindex
   - xelatex
   - latexmk
-  - xindy
 
 All of them are part of the TexLive distribution. E.g. on Debian / Ubuntu,
 install them with:
@@ -68,7 +67,7 @@ install them with:
 Or if you want to use less disk space:
 
     apt install texlive-latex-extra texlive-fonts-recommended texlive-xetex \
-                latexmk xindy fonts-freefont-otf
+                latexmk fonts-freefont-otf
 
 Compilation
 -----------

--- a/doc/sphinx/conf.py
+++ b/doc/sphinx/conf.py
@@ -356,6 +356,9 @@ latex_elements = {
 
 latex_engine = "xelatex"
 
+# Cf. https://github.com/sphinx-doc/sphinx/issues/7015
+latex_use_xindy = False
+
 ########
 # done #
 ########

